examples/basic-tracer-node/README.md

```diff
@@ -57,6 +60,22 @@ Click on the trace to view its details.
 
 <p align="center"><img src="./images/jaeger-ui-detail.png?raw=true"/></p>
 
+### Collector Exporter
+Make sure you have [docker](https://docs.docker.com/) installed
+ - Run the docker container
+   ```sh
+   $ # from this directory
+   $ # open telemetry
+   $ npm run collector:docker:ot
+   $ # or alternatively open census
+   $ npm run collector:docker:oc
+   $ # at any time you can stop & clean them
+   $ npm run collector:docker:stop
+   ```
+
+#### Collector Exporter - Zipkin UI
+Please follow the section [Zipkin UI](#zipkin-ui) only
```

Comment:
I think it worth to add a link for the [Collector](https://github.com/open-telemetry/opentelemetry-collector) and mention about we are using zipkin exporter for this example, but it is possible to change to any other exporter(s) without redeploying your application.